tachyon/init.sh

```diff
@@ -38,6 +38,15 @@ else
         wget https://s3.amazonaws.com/Tachyon/tachyon-0.6.0-cdh4-bin.tar.gz
       fi
       ;;
+    0.6.4)
+      if [[ "$HADOOP_MAJOR_VERSION" == "1" ]]; then
+        wget https://s3.amazonaws.com/Tachyon/tachyon-0.6.4-bin.tar.gz
+      elif [[ "$HADOOP_MAJOR_VERSION" == "2" ]]; then
+        wget https://s3.amazonaws.com/Tachyon/tachyon-0.6.4-cdh4-bin.tar.gz
+      else
+        wget https://s3.amazonaws.com/Tachyon/tachyon-0.6.4-hadoop2.4-bin.tar.gz
+      fi
+      ;;
```

Comment:
Could you just move the last line here to the `*)` case ?  That way hadoop2.4 should work for future Tachyon versions as well